nextcord/application_command.py

```diff
@@ -3043,6 +3078,13 @@ def slash_command(
         Permission(s) required to use the command. Inputting ``8`` or ``Permissions(administrator=True)`` for
         example will only allow Administrators to use the command. If set to 0, nobody will be able to use it by
         default. Server owners CAN override the permission requirements.
+    nsfw: :class:`bool`
+        If the command should be usable in NSFW channels or not. Setting to ``True`` will make the command
+        only usable in NSFW channels.
+        .. note::
+            Due to a discord limitation, this can only be set for the parent command for subcommands.
```

Comment:
```suggestion
        only usable in NSFW channels.

        .. note::

            Due to a discord limitation, this can only be set for the parent command for subcommands.

```